build/dockerfiles/Dockerfile

```
@@ -19,6 +19,12 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+FROM docker.io/node:lts-alpine3.14 as dwtemplates
```

```suggestion
FROM docker.io/node:16.13.0-alpine3.14 as dwtemplates
```

lts is a rolling tag and we should use only 'fixed' tag so in 6months or one year it's still using the same image